week1/labs/lab1-2c.rb

```diff
@@ -156,10 +156,14 @@
 
 def symbolist(tasks)
   new_tasks = Array.new
-  tasks.map do |k|
-    p k.keys[0].to_sym
-    p k.keys[1].to_sym
+  tasks.each do |k|
+    new_task = {}
+    k.each do |key, value|
+      new_task[key.to_sym] = value
+    end
+    new_tasks << new_task
   end
+  new_tasks
```

Comment:
nice! as usual, it's nice to compare with the map alternative (either way is fine)

```ruby
new_tasks = tasks.map {|task| symbolize(task)}
```